docs/UserGuide.md

```diff
@@ -206,7 +212,8 @@ Examples:
 
 #### 5.1.4 Link a game to a friend: `friend --link`
 
-Links a game and the associated in-game username for the game to a friend with the provided `FRIEND_ID`.
+Links a game with the specified `GAME_ID` and the associated in-game username specified by `IN_GAME_USERNAME` for the 
+game to a friend with the provided `FRIEND_ID`.
```

Comment:
Links a game with the specified `GAME_ID` and the associated in-game username specified by `IN_GAME_USERNAME` to a friend with the provided `FRIEND_ID`.